.github/workflows/ci.yml

```
@@ -50,7 +50,7 @@ jobs:
             **/dist/mf-manifest.json
             !node_modules
 
-  lint:
+  lint-npm:
     name: 'lint'
```

you should change the name also

```suggestion
name: 'lint-npm'
```